channels/ae.m3u

```diff
@@ -17,9 +17,9 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 https://admdn4ta.cdn.mgmlcdn.com/adsports4/smil:adsports4.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AjmanHD.ae" tvg-name="Ajman HD" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",Ajman HD
 http://149.100.19.226:8001/play/AJMAN_HD
-#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Kids",Al Aan (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Culture",Al Aan (720p) [Not 24/7]
```

Comment:
More like General broadcast (Lifestyle and News)
https://www.alaan.tv/about-us

Reply:
Okay. When I went to their website, I have seen they cater to young females in their culture. But do have other programming, including getting interrupted for state speeches. I changed it to General.